src/app/Main/Body/QueryPage/QueryPageContent/Parameter/QueryPage-ParameterContent.component.tsx

```diff
@@ -33,18 +84,33 @@ function createParameterList(
                 parameterLabel = t(`Subfilters.${filterElement}.name`);
             }
 
-            const parameterListItem = createParameterListItemService(
-                filterElement,
-                selectedFilters[filterElement],
-                t,
-                isSubFilter
-            );
+            const listItems: JSX.Element[] = [];
+            selectedFilters[filterElement].forEach((parameter) => {
+                const parameterListItem = createParameterName(
+                    filterElement,
+                    parameter,
+                    t,
+                    isSubFilter
+                );
+                listItems.push(
+                    <ListItem
+                        key={`listItem-${parameterListItem.key}`}
```

Comment:
Is it intentional that `keyName = parameter.replace(" ", "_");` is not used as key here anymore?